test/functional/feature_segwit.py

```diff
@@ -527,8 +527,6 @@ def mine_and_test_listunspent(self, script_list, ismine):
         elif (ismine == 1):
             assert_equal(watchcount, len(script_list))
             assert_equal(spendcount, 0)
-        else:
-            assert_equal(watchcount, 0)
         return txid
```

Comment:
What kind of case is 0?
If we can remove this, can we also remove the code that calls this method with 0?